rand.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Random;
+import java.util.Scanner;
+	public class rand {

```

Comment:
Naziv klase treba da bude velikim slovima, u ovom slucaju "rand". To se radi i zbog konveincijie i zbog kompajlera, jer meni okruzenje nije prepoznalo odmah ovo kao main metodu. Takodje bi bilo dobro da ovo bude u nekom paketu, npr. pred1zad1 ili sl. (desni klik projekat -> new package)